.gitignore

```diff
@@ -1,3 +1,6 @@
 vendor/*
 .bundle
 notes.md
+Gemfile.lock
+.ruby-version
```

Comment:
@rwdaigle you were right, copied this from https://github.com/github/gitignore/blob/master/Ruby.gitignore